### ISSUE TITLE
need logical def for "developmental pigmentation", plus related pigmentation issue

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#20843]
1. need logical def for "developmental pigmentation"

The term `regulation of developmental pigmentation` does not classify under `regulation of developmental process`. 

It looks to me like the problem is due to the fact that the term `developmental pigmentation` does not have an equivalence axiom. I don't have any idea what to suggest, any ideas @ukemi @cmungall ?

2. broaden definition of "pigmentation"?

As part of **NTR:visually-mediated background adaptation** #20524, I have recently created a term for `regulation of pigmentation`. 

Here's the current def:
term: pigmentation
def: The accumulation of pigment in an organism, tissue or cell, either by increased deposition or by increased number of cells.

For the regulation term, I added an additional phrase about modulating the distribution, not just the deposition, of the pigment 

name: regulation of pigmentation
def: "Any process that modulates the frequency, rate or extent of the deposition **_or modulates the distribution of coloring matter_** in an organism." 

Do you think we should also broaden the def of the term `pigmentation`?

**Comment by cmungall** [comment]
Issue 1: 

yes, but in fact all you need to do is add a basic subclass link between dev-pigmentation and dev. I am surprised it is not there already!

once you do that you should get the inference

![image](https://user-images.githubusercontent.com/50745/107713083-5ead6380-6c7f-11eb-9d55-001d798d144c.png)

you could certainly go one step further and add a logical definition for devpig but we don't have a design pattern for "developmental X" AFAICT

elsewhere @pgaudet suggests that devpigm may not be a very useful grouping class, I agree

Re: def of regulation term. I generally favor keeping the reg hierarchy as a dumb automatic shadow, otherwise it gets hard to maintain. I think your suggested change makes sense, but if we do this kind of thing frequently then we see drift between defs of main terms and regulation terms and we don't know which drift is accidental and which is intentional

**Comment by cmungall** [comment]
I have some questions about the pigmentation hierarchy:

![image](https://user-images.githubusercontent.com/50745/107714629-7a663900-6c82-11eb-8c00-fd87b73e0f4e.png)

 * the definition of 'pigmentation' is 'the accumulation of pigment in...`, so why is there a separate term 'pigment accumulation'?
 * the hierarchy is full of only-children, which is an antipattern
 * why is ocellus pigmentation hanging out by itself?
 * the sex-specific ones are suspect. Is there an intent to use this within a taxon? There are 2 annotations, both in fly. I looked at one and I think these terms should be made specific to cuticles, and the existing annotation should move from 'female pigmentation' to 'regulation of pigmentation of cuticle in females'
 
I'm having a hard time figuring out how we might differentiate regulation of pigmentation from pigmentation. Would it make sense to conceive of pigmentation as a regulation of biological quality (ie amount of melanin in a cell)? We avoid 2nd order regulation in the ontology, so we would collapse RoP and P - I am not sure current annotation differences are meaningful.

I might go further and discourage direct annotations for non-IMP, and encourage annotation of a mechanism - e.g an enzyme in a biosynthetic or catalobic pathway for a melanin molecule.

**Comment by ukemi** [comment]
It almost looks like the synonym and the term name for 'pigment accumulation' were accidentally swapped. Pigment accumulation is a child of response to external stimulus. It looks like the same thing happened to some of the children. See the definition cellular pigment accumulation. From the definition it looks like it should be pigment accumulation in response to external stimulus and be a sibling of the developmental term. This would require some pruning of the children. 
'The aggregation of coloring matter in a particular location in an organism, tissue or cell, occurring in response to some external stimulus.'
I would always be cautious about 'only' children and paucity of annotation as a criteria for the validity of a term. 'Only' children may be the result of lack of exhaustiveness. For example I would be willing to go out on a limb say that there are missing children from cellular pigment accumulation. 
Where do regulation and pigmentation begin and end? Good question!!!! Always a good question. I remember at one point thinking about how McClintock's Ac/Ds system would be annotated with respect to this. I like the idea of pigmentation being a regulation of a biological quality, but it should be thought about carefully. It seems that there would have to be a way to tie the qualities together when we know what chemical resulted in a certain colored pigment? I think there are some cases in plants where we still don't know. I would certainly pull in some botanists.

**Comment by krchristie** [comment]
> It almost looks like the synonym and the term name for 'pigment accumulation' were accidentally swapped. Pigment accumulation is a child of response to external stimulus. It looks like the same thing happened to some of the children. See the definition cellular pigment accumulation. From the definition it looks like it should be pigment accumulation in response to external stimulus and be a sibling of the developmental term. This would require some pruning of the children.
> 'The aggregation of coloring matter in a particular location in an organism, tissue or cell, occurring in response to some external stimulus.'
> I would always be cautious about 'only' children and paucity of annotation as a criteria for the validity of a term. 'Only' children may be the result of lack of exhaustiveness. 

I completely agree.

> For example I would be willing to go out on a limb say that there are missing children from cellular pigment accumulation.

Agreed as well.

> Where do regulation and pigmentation begin and end? Good question!!!! Always a good question. I remember at one point thinking about how McClintock's Ac/Ds system would be annotated with respect to this. I like the idea of pigmentation being a regulation of a biological quality, but it should be thought about carefully. It seems that there would have to be a way to tie the qualities together when we know what chemical resulted in a certain colored pigment? I think there are some cases in plants where we still don't know. I would certainly pull in some botanists.

Another thing to be aware of is that many vertebrates have systems of regulating external presentation of pigmentation rapidly in response to changes in background light intensity, i.e. "background adaptation", in manners that mammals appear to have lost. Ceri's requests were based on needing to annotate fish, but nal presentation of pigmentation rapidly in response to changes in background light intensity, i.e. " is found in other groups of vertebrates as well. 

I don't know exactly what is happening in fish for rapid background adaptation, but I know for sure that cephalopods can very rapidly change their presentation of pigmentation by changing the size of the pigment granule between a spread out form that displays the pigment to a compressed one that hides it. I would think that this is a regulated behavior.

**Comment by ukemi** [comment]
Yeah. The regulation question is a tough one. I think in the original terms, we kind of rolled what you are separating out as regulation into the process of pigmentation itself and the phenomena that you see in an octopus or a chameleon would all have been classified as pigmentation in response to an external stimulus at an organismal level. I think that's why developmental pigmentation and pigment accumulation (in response to external stimulus) were separated.

**Comment by ukemi** [comment]
Have you ever watched the pigment cells expand and contract in real time under the microscope? It is unbelievable cool.

**Comment by krchristie** [comment]
@pgaudet - tagging you so you can include this ticket I created and assigned myself on the rota

**Comment by krchristie** [comment]
> Have you ever watched the pigment cells expand and contract in real time under the microscope? It is unbelievable cool.

only on TV, which was still unbelievably cool
